application/src/main/java/org/togetherjava/tjbot/features/reminder/RemindRoutine.java

```diff
@@ -142,4 +136,32 @@ static ReminderRoute toPrivate(PrivateChannel channel) {
                             + " the original channel you wanted it to be send to)");
         }
     }
+
+    private void attemptRetryReminder(Reminder reminder, Throwable failure) {
+        if (reminder.pendingReminders().getFailureAttempts() > MAX_FAILURE_RETRY) {
+            logger
+                .warn("""
+                        Failed to send a reminder with (authorID '{}') skipping it. This can be due to a network issue, \
+                        but also happen if the bot disconnected from the target guild and the \
+                        user has disabled DMs or has been deleted. It's (content includes '{}' with reminderId '{}'). The reason for (failure '{}')""",
```

Comment:
The content-sentence looks and reads weird:

> It's (content includes 'hello world please remind me about foo bar blablabla haha' with reminderId '13').

Instead, go for:

> Reminder id 13, content is: hello world please remind me about foo bar blablabla haha

Reply:
having the `failure` in the message itself is incorrect. the logger will auto-attach it, dont add it to the string. just drop the last sentence, but keep it as last parameter in the method call

Reply:
ok will update this

Reply:
![image](https://user-images.githubusercontent.com/61616007/220858098-a1011051-2467-490f-9919-b4d62aafb169.png)
is this better now?

Reply:
not sure why u want to put it into `(...)`, reads weird imo. the apostrophe should be `Its´` afaik, missing space before the `(` and missing dot at the end of the sentence.

all in all, id go for

> Reminder id 13, content is: hello world please remind me about foo bar blablabla haha

Reply:
wow been using Its wrong all along, turns out It's and Its are two different things with a slight variations. Apparently It's short for `It is` or `It has`

Reply:
yeah. and if the apostrophe is put at the end, its means that u possess something, as in "that thing belongs to me".

in any case, those shortcuts generally should not be used in code. intellij also has a warning for it actually